JustSaying.Sample.Restaurant.KitchenConsole/Program.cs

```diff
@@ -0,0 +1,55 @@
+using Amazon;
+using JustSaying.Sample.Restaurant.Models;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Threading.Tasks;
+
+namespace JustSaying.Sample.Restaurant.KitchenConsole
+{
+    internal class Program
+    {
+        private static async Task Main()
+        {
+            Console.Title = "KitchenConsole";
+
+            await new HostBuilder()
+               .ConfigureLogging((hostContext, logging) =>
+               {
+                   logging.AddConsole();
+               })
+              .ConfigureServices((hostContext, services) =>
+              {
+                  services.AddJustSaying(config =>
+                  {
+                      config.Client(x =>
+                      {
+                          // Use the simulated pafortin/goaws docker image
+                          x.WithServiceUrl("http://localhost:4100");
+
+                          // or supply credentials if using the real aws services
+                          //x.WithCredentials(...)
+                      });
+                      config.Messaging(x =>
+                      {
+                          x.WithRegion(RegionEndpoint.APSoutheast2);
```

Comment:
Make this a config value? For example, we primarily use EU West 1.

Reply:
done